security-policies/bundle/compliance/policy/azure/sql_server/ensure_vulnerability_assessment_storage_account.rego

```diff
@@ -0,0 +1,10 @@
+package compliance.policy.azure.sql_server.ensure_vulnerability_assessment_storage_account
+
+import future.keywords.if
+
+import future.keywords.every
+
+ensure_vulnerability_assessment_storage_account(va) if {
+	not va.storageAccountName == null
+	not trim(va.storageAccountName, " ") == ""
```

Comment:
why are we trimming spaces? Would only spaces be a valid storage account name?

Reply:
I did out of precaution.... Because the storageAccountName is being fetched based on string manipulation (https://github.com/elastic/cloudbeat/pull/1824/files#diff-d175b30a8a978a9032c576340ec45c3f5eedce63fbfbcaef4325c2c3925028c5R348) I thought it was a good idea. But maybe it's being overly cautious.